app/services/StoreDataService.scala

```diff
@@ -84,10 +84,17 @@ class StoreDataServiceImpl extends StoreDataService {
     AboutYou(
       fullName = userAnswers.get(WhatIsYourFullNamePage),
       telephoneNumber = userAnswers.get(YourPhoneNumberPage),
-      doYouHaveAEmailAddress = userAnswers.get(DoYouHaveAnEmailAddressPage),
       emailAddress = userAnswers.get(YourEmailAddressPage),
       dateOfBirth = userAnswers.get(WhatIsYourDateOfBirthPage),
       mainOccupation = userAnswers.get(WhatIsYourMainOccupationPage),
+      contactPreference = userAnswers.get(HowWouldYouPreferToBeContactedPage) match {
+        case Some(preference) => val contactPreference: Set[Preference] = preference.map{
+          case HowWouldYouPreferToBeContacted.Email => Email
+          case HowWouldYouPreferToBeContacted.Telephone => Telephone
+        }
+        Some(ContactPreferences(contactPreference))
```

Comment:
Could we move the above val onto its own line and then indent here please? So that it's a bit clearer.